### PR TITLE
apply preferred style to com.jme3.renderer

### PR DESCRIPTION
**Pull request by stephengold**
This PR illustrates minimal changes to make the `com.jme3.renderer` package comply with the coding style proposed at issue #1098.

Marked as "draft" because the PR includes 2 API changes that should be integrated separately, as discussed at #1098.

### CODE REVIEW

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java [hunk 1]
```diff
@@ -1480,13 +1555,17 @@ public Vector3f getScreenCoordinates(Vector3f worldPosition, Vector3f store) {
     }
 
     /**
+     * Determines the display width.
```
Comment (Scrappers-glitch) [inline jme3-core/src/main/java/com/jme3/renderer/Camera.java:1558]: Should we change this to `Gets` or `Retrieves`.

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java [hunk 2]
```diff
@@ -1480,13 +1555,17 @@ public Vector3f getScreenCoordinates(Vector3f worldPosition, Vector3f store) {
...
      * @return the width/resolution of the display.
      */
     public int getWidth() {
         return width;
     }
 
     /**
+     * Determines the display height.
+     *
      * @return the height/resolution of the display.
```
Comment (Scrappers-glitch) [inline jme3-core/src/main/java/com/jme3/renderer/Camera.java:1569]: And this too.

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java (hunk 1 above)
Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:1558]: The choice between "Gets" and "Determines" isn't addressed in the style guide.

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java (hunk 2 above)
Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:1569]: The choice between "Gets" and "Determines" isn't addressed in the style guide.

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java (hunk 1 above)
Comment (Scrappers-glitch) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:1558]: Yes, but determines is a little bit confusing for a getter method, unless this getter is internally changing code.

Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:1558]: How about "Returns"?

Comment (Scrappers-glitch) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:1558]: I think its better than determines.

File: jme3-core/src/main/java/com/jme3/renderer/Camera.java [hunk 3]
```diff
@@ -485,7 +483,7 @@ public void resize(int width, int height, boolean fixAspect) {
     }
 
     /**
-     * <code>getFrustumBottom</code> returns the value of the bottom frustum
+     * Returns the value of the bottom frustum
```
Comment (Sailsman63) [inline jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: Completely aside from the format/styling concerns, I question the value of this, and other "Trivial" Javadoc comments in this file. It adds no details that cannot be gleaned from the method signature, and therefore does not add value above what the javadoc tool would create on it's own. Why does it exist?

Comment (Sailsman63) [inline jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: Completely aside from the format/styling concerns, I question the value of this, and other "Trivial" Javadoc comments in this file. It adds no details that cannot be gleaned from the method signature, and therefore does not add value above what the javadoc tool would create on it's own. Why does it exist?

Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: The style guide says that javadoc is optional for "simple, obvious" methods like getFoo, in cases where there really and truly is nothing else worthwhile to say but "Returns the foo".

Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: This PR show minimal changes. Deleting the entire javadoc seemed like a bigger change than deleting one word.

I agree that the current javadoc isn't valuable. Honestly, I haven't studied the frustum math to figure out what "the value of the bottom frustum" really is. Is it the y/z slope, the y-coordinate at the near plane, or the y-coordinate at the far plane? So I believe `getFrustumBottom()` needs *some* javadoc, but I don't yet know enough to write it myself. Writing javadoc for someone else's code can be very frustrating!

Comment (Sailsman63) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: Good point, though in this case the code literally passes through the value of a field.

I hear you on documenting other people's code. In the case of the frustrum stuff here, I'd think that the javadoc should go on the field declarations, as they are `protected` and could, in theory, be overridden by extension code.

Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: In theory, yes.

Comment (stephengold) [reply jme3-core/src/main/java/com/jme3/renderer/Camera.java:521]: Ok, there's a proposed change (to the style guide) that would remove the option of omitting javadoc for "simple, obvious" methods. Discussion of this proposed change is at #1098.

**Comment by stephengold** [comment]
I believe this PRs usefulness as a conversation piece is over. I'll revert the 2 API changes and make another pass over the files.

**Comment by stephengold** [comment]
I plan to integrate this in 24 hours.

**Pull request merged**